app/src/main/java/com/sample/appconnectsample/ListActivity.java

```diff
@@ -74,15 +75,27 @@ private void populateForms() {
         // studies. Here we just gather all available forms, but you could also
         // present them organized by subject if desired.
         Datastore datastore = App.getUIDatastore(this);
-        for (Subject subject : datastore.getSubjectsForUser(userID))
+        for (Subject subject : datastore.getSubjectsForUser(userID)) {
+            if (subjectID == 0)
```

Comment:
It's technically possible that a subjectID could be zero - could we set it to `-1` above and check that?